<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileWindow</class>
 <widget class="QMainWindow" name="ProfileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profile</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="UsernameTxt">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dummy User </string>
    </property>
   </widget>
   <widget class="QLabel" name="UseremailTxt">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dummy@email.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="UsernumberTxt">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0XX--xxxxxxx</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditInfoBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit info</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>631</width>
      <height>521</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Over View</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>581</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Past sale graph</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>349</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Bought graph</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>350</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Profit</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>350</y>
        <width>181</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Data Graph</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>History</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>20</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sell</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bought</string>
       </property>
      </item>
     </widget>
     <widget class="QTableWidget" name="historyTable">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>120</y>
        <width>501</width>
        <height>341</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Date Upload</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="Wallet">
     <attribute name="title">
      <string>Wallet</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>230</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add or update credit card info</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="BackBtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>520</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
